test/integ-new/main_new_test.go

```diff
@@ -180,6 +182,41 @@ func TestAPIAndPublicKeyService(t *testing.T) {
 	}
 }
 
+func TestUIService(t *testing.T) {
+	t.Parallel()
+
+	addr := cfg.UIServiceAddr
+	wantStatusCode := http.StatusOK
+	endpoint := "/health"
+
+	t.Run("test_health_checkpoint", func(t *testing.T) {
+		ctx := context.Background()
+
+		client := &http.Client{
+			Timeout: 5 * time.Second,
+		}
+
+		uri := fmt.Sprintf("%s%s", addr, endpoint)
+		req, err := http.NewRequestWithContext(ctx, "GET", uri, nil)
+		if err != nil {
+			t.Fatalf("failed to create request: %v", err)
+		}
+
+		req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", cfg.UIServiceIDToken))
+
+		resp, err := client.Do(req)
+		if err != nil {
+			t.Fatalf("client failed to get response: %V", err)
+		}
+
+		defer resp.Body.Close()
+
+		if diff := cmp.Diff(wantStatusCode, resp.StatusCode); diff != "" {
+			t.Errorf("response status code got unexpected diff (-want, +got):\n%s", diff)
+		}
```

Comment:
Should be able to do `if got, want := resp.StatusCode, wantStatusCode; got != want { ... }`